apps/extension/src/clients/index.ts

```diff
@@ -1,12 +0,0 @@
-import { createPromiseClient } from '@connectrpc/connect';
-import { ViewProtocolService } from '@buf/penumbra-zone_penumbra.connectrpc_es/penumbra/view/v1alpha1/view_connect';
-import { createChannelTransport, ChannelClientLabel } from '@penumbra-zone/transport';
-import { ClientConnectionManager } from '@penumbra-zone/transport/src/chrome-runtime/client-connection-manager';
-
-export const grpcClient = createPromiseClient(
-  ViewProtocolService,
-  createChannelTransport(
-    ViewProtocolService,
-    ClientConnectionManager.init(ChannelClientLabel.Extension),
-  ),
-);
```

Comment:
It looks like you found a workaround, but this should be able to work. There is something deeper wrong. Can you do more debugging to see why the popup is not able to use this grpc client?

Reply:
Ok

Reply:
sorry, this looks like my fault. missed this internal client init in one of the refactors during review. intra-extension transport init requires one more step. i'll correct this in a moment